spec/controllers/shared_context.rb

```diff
@@ -0,0 +1,7 @@
+shared_context 'Authenticated User' do
```

Comment:
The name of context should be named starting with `when with or without`
```suggestion
shared_context 'with authenticated user' do
```